client/gutenberg-editor/main.jsx

```diff
@@ -0,0 +1,13 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+
+class GutenbergEditor extends Component {
+	render() {
+		return <div className="gutenberg-editor">Hello, Gutenberg!</div>;
+	}
+}
+
+export default GutenbergEditor;
```

Comment:
directory structure seems reasonable. the other idea is to bundle all Gutenberg-related content inside `client/gutenberg` making this `client/gutenberg/editor` and others `client/gutenberg/extensions` or something like that.

Reply:
Thanks updated in 45b203b!